croptalk/document_retriever.py

```diff
@@ -94,6 +117,12 @@ def _format_query_response(query_response: QueryReturn) -> List[str]:
             f" county='{doc.properties['county']}'"
             f" s3_key='{doc.properties['s3_key']}'"
             f" url='https://croptalk-spoi.s3.us-east-2.amazonaws.com/{doc.properties['s3_key']}'"
-            f">{doc.properties['content']}</doc>"
             for i, doc in enumerate(query_response.objects)
         ]
+
```

Comment:
It may return the same doc multiple times, let's add a validation that it has not been added yet

Reply:
Ok solved ! I made sure to only remove SP documents which are return multiple times